### PR TITLE
Improve descriptiveness of elements

### PR DESCRIPTION
**Pull request by lisicase**
- Added descriptions of diagrams (addresses accessibility work item [2117264](https://identitydivision.visualstudio.com/OData/_workitems/edit/2117264))
  - Added brief informative descriptions of images in alt text
  - Changed structure of elements to use the <figure/> and <figcaption/> tags to allow for more complete, informative descriptions of the diagrams
- Removed unnecessary links to a page dedicated to an individual image (observation in accessibility work item [2117264](https://identitydivision.visualstudio.com/OData/_workitems/edit/2117264) no longer relevant)
- Added aria-labels for screen readers that describe what the PDF links link to (addresses accessibility work item [2115630](https://identitydivision.visualstudio.com/OData/_workitems/edit/2115630))

### CODE REVIEW

File: public/css/site.css [hunk 1]
```diff
@@ -380,3 +380,14 @@ strong {
   font-weight: bold;
   font-size: 1.5rem;
 }
+
```
Comment (xuzhg) [inline public/css/site.css:383]: where is it used?

**Review by xuzhg — APPROVED** [review]

File: pages/documentation/documentation.html [hunk 2]
```diff
@@ -8,15 +8,15 @@ <h2>OData Version 4.01</h2>
...
+<li><a href="http://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html">OData JSON Format</a> | <a href="http://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.pdf" aria-label="'OData JSON Format' PDF">PDF</a></li>
+<li><a href="http://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html">OData Common Schema Language (CSDL) XML Representation</a> | <a href="http://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.pdf" aria-label="'OData Common Schema Language (CSDL) XML Representation' PDF">PDF</a></li>
+<li><a href="http://docs.oasis-open.org/odata/odata-csdl-json/v4.01/odata-csdl-json-v4.01.html">OData Common Schema Language (CSDL) JSON Representation</a> | <a href="http://docs.oasis-open.org/odata/odata-csdl-json/v4.01/odata-csdl-json-v4.01.pdf" aria-label="'OData Common Schema Language (CSDL) JSON Representation' PDF">PDF</a></li>
```
Comment (mikepizzo) [inline pages/documentation/documentation.html:16]: It seems like all but the first label ends in "PDF". We should probably be consistent. I don't have a strong feeling either way; just that we should be consistent.

**Review by mikepizzo — APPROVED** [review]
Approved with minor comment (label in line 11 of documentation.html should  include "PDF")

File: pages/documentation/documentation.html (hunk 2 above)
Comment (lisicase) [reply pages/documentation/documentation.html:16]: Good catch, thanks!

File: public/css/site.css (hunk 1 above)
Comment (lisicase) [reply public/css/site.css:383]: This class is used to visually hide the screen reader descriptions of the diagrams, so they can be listened to but not seen.

File: pages/documentation/documentation.html [hunk 3]
```diff
@@ -8,15 +8,15 @@ <h2>OData Version 4.01</h2>
...
 <li><a href="http://docs.oasis-open.org/odata/odata/v4.01/cs01/abnf/odata-abnf-construction-rules.txt">OData ABNF</a></li>
```
Comment (mikepizzo) [inline pages/documentation/documentation.html:17]: Note we're still a bit inconsistent -- with the first labels, the closing single quote is before the PDF. In the later labels, the PDF is outside of the single quotes.

**Pull request merged**